### PR TITLE
Deploying heroku

### PR DESCRIPTION
**Pull request by hshahwan**
adding `   "heroku-postbuild": "npm run client:init && npm run client:build"` to package.json.
 update style names and some style

### CODE REVIEW

File: server/database/db_connection.js [hunk 1]
```diff
@@ -1,7 +1 @@
-const { Pool } = require('pg');
-require('env2')('./config.env');
 
-if (!process.env.DB_URL) throw new Error('DB_URL is not set!');
```
Comment (sultanassi95) [inline server/database/db_connection.js]: why did u delete that?

File: server/database/db_build.js [hunk 2]
```diff
@@ -1,17 +1 @@
...
-const fs = require('fs');
 
-const sql = fs.readFileSync(`${__dirname}/db.sql`).toString();
-const dbConnect = (cb) => {
-    dbcon.query(sql, (err, res) => {
-        if (err) return cb(err);
-        cb(null, res);
-    });
-};
-
```
Comment (sultanassi95) [inline server/database/db_build.js]: why did u delete that file?

**Comment by m4v15** [comment]
@hshahwan does this work on heroku with these changes now? If so, maybe use this branch to link to the live app?

**Comment by RamyAlshurafa** [comment]
keep the Branch

**Pull request closed without merging**